.changeset/chilled-trains-laugh.md

```diff
@@ -9,7 +9,16 @@
 '@graphiql/toolkit': minor
 ---
 
-_BREAKING CHANGE:_ drop commonjs exports in all libraries except for `graphiql` and `@graphiql/react`
+_BREAKING CHANGE:_ drop commonjs exports in all libraries except for `graphiql`
+and `@graphiql/react`
```

Comment:
why is this on a new line? it's harder to read

Reply:
Some locale prettier changes, will revert 👍